src/db/fraudstersData.js

```diff
@@ -1,48 +1,68 @@
-import {ObjectId} from 'mongodb';
-import {validateId} from '../validations/Validations.js';
-import{fraudsters} from '../config/mongoCollections.js';
-import{reports} from '../config/mongoCollections.js';
-import{users} from '../config/mongoCollections.js';
+import { ObjectId } from 'mongodb';
+import { validateId } from '../validations/Validations.js';
+import { fraudsters } from '../config/mongoCollections.js';
+import { reports } from '../config/mongoCollections.js';
+import { users } from '../config/mongoCollections.js';
 import * as validations from '../validations/Validations.js';
 import * as reportsData from './reportsData.js';
-import {AlertService} from '../services/AlertService.js'
+import { AlertService } from '../services/AlertService.js'
 
 export const getFraudsterById = async (fraudsterId) => {
-    fraudsterId = validations.validateId(fraudsterId);
-    const fraudsterCollection = await fraudsters();
-    const fraudster = await fraudsterCollection.findOne({_id: new ObjectId(fraudsterId)});
-    if(!fraudster) throw new Error (`fraudster not found`);
-    return fraudster;
+  fraudsterId = validations.validateId(fraudsterId);
+  const fraudsterCollection = await fraudsters();
+  const fraudster = await fraudsterCollection.findOne({ _id: new ObjectId(fraudsterId) });
+  if (!fraudster) throw new Error(`fraudster not found`);
+
+  if (await isFraudsterTrending(fraudsterId)) {
+    updatedFraudster = await fraudsterCollection.findOneAndUpdate({ _id: new ObjectId(fraudsterId) }
```

Comment:
is this required for  the getFraudsterById function? 
Also updatedFraudster is not defined (need to include let or const)